api/IDBFactory.json

```
@@ -210,6 +201,56 @@
           }
         }
       },
+      "databases": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBFactory/databases",
+          "support": {
+            "chrome": {
+              "version_added": "71"
+            },
+            "chrome_android": {
+              "version_added": "71"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/934640'>bug 934640</a>."
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/934640'>bug 934640</a>."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "58"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
```

The latest [Samsung Internet version](https://github.com/mdn/browser-compat-data/blob/master/browsers/samsunginternet_android.json#L132) is based off of Chrome 67, so this should be `false`.

```suggestion
              "version_added": false
```